source/Calamari/Deployment/SpecialVariables.cs

```diff
@@ -134,14 +130,12 @@ public static class Azure
                 public static readonly string SwapIfPossible = "Octopus.Action.Azure.SwapIfPossible";
                 public static readonly string StorageAccountName = "Octopus.Action.Azure.StorageAccountName";
                 public static readonly string UseCurrentInstanceCount = "Octopus.Action.Azure.UseCurrentInstanceCount";
-
                 public static readonly string UploadedPackageUri = "Octopus.Action.Azure.UploadedPackageUri";
-
                 public static readonly string CloudServicePackagePath = "Octopus.Action.Azure.CloudServicePackagePath";
-
-                public static readonly string CloudServicePackageExtractionDisabled = "OctopusCloudServicePackageExtractionDisabled";
-
-                public static readonly string LogExtractedCspkg = "Octopus.Action.Azure.LogExtractedCspkg"; 
+                public static readonly string PackageExtractionPath = "Octopus.Action.Azure.PackageExtractionPath";
+                public static readonly string CloudServicePackageExtractionDisabled = "Octopus.Action.Azure.CloudServicePackageExtractionDisabled";
+                public static readonly string LogExtractedCspkg = "Octopus.Action.Azure.LogExtractedCspkg";
+                public static readonly string CloudServiceConfigurationFileRelativePath = "Octopus.Action.Azure.CloudServiceConfigurationFileRelativePath";
```

Comment:
I think perhaps we should leave the name of this variable as it was, for backwards compatibility and documentation consistency?